src/test/java/com/egov/core-services/tests/userUpdate.feature

```diff
@@ -0,0 +1,90 @@
+        * print updatedUserprofileResponseBody.Errors[0].message
+        * assert updatedUserprofileResponseBody.Errors[0].message == errorMessage.errorMessages.invalidPhoneNumberError
+
+@User_Update_RandomDigit_MobileNumber_06  @negative @user
+Scenario: To verify the phone number is not get updated even though it is valid
+        * call read('../pretests/updateUserPretest.feature@validRandomPhoneNumber')
+        * print updatedUserprofileResponseBody.user[0].mobileNumber
+        * assert updatedUserprofileResponseBody.user[0].mobileNumber != userProfileData.inValidProfileData.mobileNumber 
+
+@User_Update_InvalidGender_07  @negative    @user
+Scenario: To verify the error message returned by API when an invalid gender is passed
+        * call read('../pretests/updateUserPretest.feature@invalidGender')
+        * print updatedUserprofileResponseBody.Errors[0].message
+        * assert updatedUserprofileResponseBody.Errors[0].message == errorMessage.errorMessages.invalidGenderError  
+
+@User_Update_noTenantId_08  @negative   @user
+Scenario: To verify the error message returned by API when tenant id passed as null
+        * call read('../pretests/updateUserPretest.feature@invalidTenantId')
+        * print updatedUserprofileResponseBody.Errors[0].message
+        * assert updatedUserprofileResponseBody.Errors[0].message == errorMessage.errorMessages.tenantIdNull 
+
+@User_Update_noUUID_09  @negative   @user
+Scenario: To verify the error message returned by API when UUID passed as blank
+        * call read('../pretests/updateUserPretest.feature@blankUUID')
+        * print updatedUserprofileResponseBody.Errors[0].code
+        * assert updatedUserprofileResponseBody.Errors[0].code == errorMessage.errorMessages.noUUIDErrorCode
+
+@User_Update_InvalidUUID_10  @negative  @user
+Scenario: To verify the error message returned by API when an invalid UUID passed
+        * call read('../pretests/updateUserPretest.feature@invalidUUID')
+        * print updatedUserprofileResponseBody.Errors[0].code
+        * assert updatedUserprofileResponseBody.Errors[0].code == errorMessage.errorMessages.noUUIDErrorCode 
+
+@User_Update_noID_11  @negative @user
+Scenario: To verify the error message returned by API when ID is removed
+        * call read('../pretests/updateUserPretest.feature@removeID')
+        * print updatedUserprofileResponseBody.Errors[0].message
+        * assert updatedUserprofileResponseBody.Errors[0].message == errorMessage.errorMessages.tenantIdNull  
+
+@User_Update_InvalidID_12  @negative    @user
+Scenario: To verify the error message returned by API when ID is invalid
+        * call read('../pretests/updateUserPretest.feature@invalidID')
+        * print updatedUserprofileResponseBody.Errors[0].message
+        * assert updatedUserprofileResponseBody.Errors[0].message == errorMessage.errorMessages.tenantIdNull
+
+@User_Update_NameWithMorethanMaxCharacters_13  @negative    @user
```

Comment:
please keep uniform spacing between multiple tags. please apply this in all places